static/mainnet/defactor-pools.json

```diff
@@ -57,5 +57,25 @@
         "collateral": "0x6fC27F5CC0aAFeC8e2b8bC4E6393aC89e45232d3"
       }
     ]
+  },
+  {
+    "id": "pools-landshare-mainnet",
+    "name": "LandShare Production",
+    "description": "Production environment for LandShare.",
+    "web": "https://landshare.factr.app",
+    "api": "https://landshare-api.factr.app",
+    "chains": [
+      {
+        "id": "Bsc",
+        "type": "Erc20 Collateral Pool",
+        "contract": "0x6cbA45B0228F7e2e058e877060619225F9979c87",
+        "token": "0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d",
+        "oracle": "0x60E87395b0101F85C623c38Be010574f958496db",
```

Comment:
@leisterfrancisco  is this the correct value for oracle?

Reply:
No, this is the right value.

https://bscscan.com/address/0xd2896dE7De086BFcd80B66356458Ae61BE4aBEaD